src/transformers/models/cpmant/configuration_cpmant.py

```diff
@@ -0,0 +1,125 @@
+# coding=utf-8
+# Copyright 2022 The OpenBMB Team and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" CPMAnt model configuration"""
+
+from typing import List, Optional, Tuple
+
+import torch
```

Comment:
here you need to add something like 
```
if torch_is_available():
     import torch
```
otherwise the tests will fail as you can see from the CI results

Reply:
Even better, can you maybe remove it? Check my comment below